src/header.rs

```diff
@@ -63,21 +62,6 @@ impl PartialEq<TagType> for TagType {
     }
 }
 
-impl PartialOrd<u32> for TagType {
```

Comment:
This was unused and is not required at all. I added this a few PRs earlier by more or less by accident.